guides/common/modules/proc_installing-the-project-ansible-modules.adoc

```diff
@@ -1,12 +1,30 @@
 [id="Installing_the_Project_Ansible_Modules_{context}"]
 = Installing the {Project} Ansible modules
 
-Use this procedure to install the {Project} Ansible modules.
+Before you can start using modules from {Project} Ansible Collections, you must install the package that provides them.
+Install the package on the system where you will be executing your playbooks.
+This system must be able to reach the {Project} API.
+It can be {ProjectServer} itself.
+ifdef::satellite[]
```

Comment:
Why the `ifdef`?

Reply:
Because for Satellite, the command to install on Satellite server uses satellite-maintain but if you install on any other system that can reach the API, it will just be dnf. So I added that ifdef statement to make it clear that this particular command is the one for Satellite server.

And now that I wrote it down like this, maybe I should mention both commands and explain all of this, rather than play with ifdefs.

Reply:
ah!

now that I am thinking of it, I think on a Satellite this is installed by default, but didn't verify

Reply:
Um, I think I had to install the package on my Satellite but I'll check again with a fresh one.

Reply:
No, it looks like you're right and the package is installed on a Satellite by default. That means I should exclude the whole procedure. Or replace it with some sort of explanation about the package for Satellite builds.

Reply:
I updated the procedure to reflect that on a Satellite server, the package is installed by default. As a result, for Satellite builds, the procedure now talks about installing the package on a system other than a server.